### DISCUSSION TITLE
Losing rank and PP after submiting lazer scores

### DISCUSSION THREAD

**Discussion by GSculerlor** [ppy/osu#27223] [Q&A]
to put it in context, I haven't submitted any stable score for years if I remember correctly and a couple of days ago, decided to try lazer score submission. after submitting my first score, my rank got dropped and pp somehow showed as 0 (both on 4K and 7K) but the overall PP still showed. I thought this was caused by the score still being processed or indexed on the score submission process but it has been more than a week and somewhat rank still dropped and pp still showed up as 0. do I need to submit stable score to trigger something on the score processor?

also to put more context, submitted scores are behave correctly as far as I can tell (visible on the beatmap leaderboard and my top scores, unlike other reported issue that the score is not visible on some page)

profile in context: https://osu.ppy.sh/users/5011137

(also noticed a problem when achieving high rank lazer score but seems like it already tracked on https://github.com/orgs/ppy/projects/19/views/1?pane=issue&itemId=51709423)

already looked up similar issues or discussions with the keyword `score submission` but seems like none of them are similar to this issue (at least on my quick search)

**Comment by Rekunan** [comment]
I do know that there are some mania pp changes merged in lazer that haven't been fully pushed to live/stable yet, maybe that's why?
